SW-111 * create-item page

ChristofferGamel opened the pull request:
perhaps attributterne skal kunne redigeres? (tilføj/fjern attributter)

ChristofferGamel committed:
formfields for create item

ChristofferGamel committed:
cleanup

ChristofferGamel committed:
removed redundant ngIf's

Thekid2002 commented on src/app/warehouse/create-item/create-item.component.ts:
```diff
@@ -9,14 +9,43 @@ import { HttpClient } from '@angular/common/http';
   constructor(private http: HttpClient) {}
+  fileName = '';
+
+  // eslint-disable-next-line @typescript-eslint/typedef
```
Fjern kommenteret funktion

Thekid2002 commented on src/app/warehouse/create-item/create-item.component.ts:
```diff
@@ -9,14 +9,43 @@ import { HttpClient } from '@angular/common/http';
+    suitables: new FormControl(),
+    wineType: new FormControl()
   });
-
```
Der må gerne være en linje plads her

Thekid2002 commented on src/app/warehouse/create-item/create-item.component.ts:
```diff
@@ -9,14 +9,43 @@ import { HttpClient } from '@angular/common/http';
+  suitables = new FormControl('');
+  suitablesList: string[] = ['Dessert', 'Fisk', 'Havregrød'];
+
+
```
her behøver der ikke være 2 linjers plads

Thekid2002 commented on src/app/webshop/catalogue/catalogue.component.ts:
```diff
@@ -185,12 +185,12 @@ export class CatalogueComponent implements OnInit {
-    case 'high-to-low':
-      this.displayWines = this.displayWines.sort((a, b) => b.price - a.price);
-      break;
+      case 'low-to-high':
```
Tror ikke den her fil skal ændres i :D

Thekid2002 commented on src/app/warehouse/create-item/create-item.component.ts:
```diff
@@ -9,14 +9,43 @@ import { HttpClient } from '@angular/common/http';
+  suitablesList: string[] = ['Dessert', 'Fisk', 'Havregrød'];
+
+
   itemForm = new FormGroup({
```
Brug formbuilder i stedet :D Der er et par eksempler i koden

Thekid2002 commented on src/app/warehouse/create-item/create-item.component.html:
```diff
@@ -1,25 +1,86 @@
+          <mat-option>None</mat-option>
+          <mat-option value="wine">Wine</mat-option>
+          <mat-option value="liquor">Liquor</mat-option>
+          <mat-option value="option3">yes</mat-option>
```
yes? der skal nok også tilføjes en DefaultItem option

Thekid2002 commented on src/app/warehouse/create-item/create-item.component.html:
```diff
@@ -1,25 +1,86 @@
+
+        <mat-form-field>
+          <mat-label>Vintype</mat-label>
+          <mat-select [(value)]="wineType">
```
Her må du kunne ngfor på itemtype

Thekid2002 reviewed (changes requested):
Husk at tilføje et billede af hvordan det ser ud på pull requests i front end

ChristofferGamel committed:
Merge branch 'develop' into SW-111

ChristofferGamel committed:
form submit setup, needs post

ChristofferGamel committed:
Merge branch 'develop' into SW-111

ChristofferGamel committed:
Submit form set up, needs a function for post

ChristofferGamel committed:
url change

ChristofferGamel committed:
added error response

ChristofferGamel committed:
enum load

Thekid2002 committed:
SW-11 * Changed ALOT

Thekid2002 committed:
Merge branch 'develop' into SW-111

Thekid2002 committed:
SW-111 * Added fields to item, fixed merge conflict

ChristofferGamel committed:
changed values

ChristofferGamel committed:
string to number on form

Commit:
Merge branch 'develop' into SW-111

Commit:
SW-111 * Create item now asks for all relevant information

And also created enums for LiqourType and SuitableFor

Commit:
SW-111 * edit item site startet WIP

Doesn't work on the create -item part where i want to edit an item from it's id. The routing works, need the implementation part of actually editing

Commit:
SW-111 * Edit items now show correct data from id from db

Thekid2002 committed:
SW-111
